File: app/src/main/java/com/gmail/jorgegilcavazos/ballislife/features/boxscore/BoxScoreView.java
package com.gmail.jorgegilcavazos.ballislife.features.boxscore;

import com.gmail.jorgegilcavazos.ballislife.features.model.BoxScoreValues;

public interface BoxScoreView {

    void showVisitorBoxScore(BoxScoreValues values);

    void showHomeBoxScore(BoxScoreValues values);

    void setLoadingIndicator(boolean active);

    void hideBoxScore();

    void showLoadingBoxScoreErrorMessage();

    void showBoxScoreNotAvailableMessage();

    void hideLoadMessage();
}

File: app/src/main/java/com/gmail/jorgegilcavazos/ballislife/features/boxscore/BoxScoreView.kt
package com.gmail.jorgegilcavazos.ballislife.features.boxscore

import com.gmail.jorgegilcavazos.ballislife.features.model.BoxScoreValues

interface BoxScoreView {

	fun showVisitorBoxScore(values: BoxScoreValues)

	fun showHomeBoxScore(values: BoxScoreValues)

	fun setLoadingIndicator(active: Boolean)

	fun hideBoxScore()

	fun showBoxScoreNotAvailableMessage(active: Boolean)

	fun showUnknownErrorToast(code: Int)
}
